cmd/helm/root.go

```diff
@@ -315,13 +315,18 @@ Common actions for Helm:
 - helm install:   upload the chart to Kubernetes
```

Comment:
Since you're cleaning up the help message, how about replacing helm fetch with helm pull right above here? Must be left over from helm v2.

Reply:
good catch. Will fix